clients/drcachesim/analyzer_multi.cpp

```diff
@@ -174,9 +194,49 @@ analyzer_multi_t::analyzer_multi_t()
 
 analyzer_multi_t::~analyzer_multi_t()
 {
+
+#ifdef HAS_ZIP
+    if (!op_record_file.get_value().empty()) {
```

Comment:
Do we really need a record-replay mode for the analyzer?

record-replay was useful for the simulator because there may be factors outside the scheduler's control, introduced by the simulator. But here, are there any such factors?

Reply:
Yes: a primary goal is to run an analysis tool on the same schedule we just ran a simulator on.